views/components/BetaSignUpForm.js

```diff
@@ -1,63 +1,62 @@
 import React from 'react';
 import FormField from './FormField';
 
-const BetaSignUpForm = props => {
+const BetaSignUpForm = ({ optin, betaCount }) => {
+  let fieldViews = [];
+  let defaultBetaCount = 3;
```

Comment:
A minor thing, but we can make this `const` since it's a value that shouldn't change.